Tasks/NpmAuthenticate/task.json

```diff
@@ -0,0 +1,102 @@
+{
+    "id": "ad884ca2-732e-4b85-b2d3-ed71bcbd2788",
+    "name": "npmAuthenticate",
+    "friendlyName": "npm Authenticate",
+    "description": "Authenticate a npmrc for tasks which use npm.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613746)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "demands": [
+        "npm"
```

Comment:
I don't think we're really using npm for anything in the auth task, so this can be removed.